db/schema.rb

```diff
@@ -26,7 +26,7 @@
     t.datetime "updated_at"
     t.string   "control_message"
     t.string   "activity"
-    t.string   "time_zone",       limit: 255, default: "UTC"
+    t.string   "time_zone",       limit: 255, default: "Pacific Time (US & Canada)"
```

Comment:
Leftover from un-run migration from @x41farmer that got merged in.  No harm in leaving it there for now until time zones/user preferences is working.